### DISCUSSION TITLE
Can the lyrics window have the ability to anchor to the player?

### DISCUSSION THREAD

**Discussion by Logansfury** [audacious-media-player/audacious#1429] [Ideas]
Hello,

I am enjoying the Audacious 4.4 but I have a suggestion. The player, EQ and playlist editer can all anchor to each other and all three move together when the player is moved. The new Lyrics window however, is "free-floating" I would love to be able to move the player alongside or above (preferably both) and have the lyrics window anchor to it, and begin moving along with it.

Thank you for reading,

Logan

**Comment by radioactiveman** [comment]
You are probably using the Winamp interface, right? Here docking plugin windows is not supported.
This is only possible with the Qt or GTK interface.

**Comment by Logansfury** [comment]
Hello Thomas and thank you very much for getting back to me.

I am indeed using the winamp interface with an absolutely beautiful skin
called DeadScarlet. It will be a shame to not see that skin any longer >.<

Switching to Qt I see that the lyrics window appears and has a button to be
shown inside the player. This isn't what I was hoping for. That narrow
window displays very little of the lyrics. I usually have the lyric window
in a long and narrow vertical display and this is what I would like to
position alongside the player and have follow onscreen.

Is it possible to get anchoring ability in addition to embedding it in the
player window?

Also the mpris2 plugin doesnt seem to work on 4.4. I use an app to display
currently playing song name, album name, artist name and cover art from
mpris output. v4.1 populates these fields and displays correct album art.
v4.4 leaves the app completely blank.

Thank you for reading!

Logan

**Comment by radioactiveman** [comment]
The plugin windows can be resized and their size is also saved to remember their size after a restart. And if the window is docked, you can also resize it to show more of the lyrics. I don't see what's missing here.

![Lyrics](https://github.com/user-attachments/assets/4911cb6b-e532-4be2-8d94-ab8d69fd7180)

Regarding mpris2: It works as expected, see this screenshot of Ubuntu with the audio controls and `playerctl` on the command line.
Make sure to have the Audacious mpris2 plugin enabled and also check other players. You didn't state which app you use, but maybe you should report it to them, not us. :)

![mpris2](https://github.com/user-attachments/assets/f6b93fd5-0e41-487c-b0b6-5639b7f0155d)

**Comment by Logansfury** [comment]
I like the configuration from the Lyrics.png very much but I cannot seem to
duplicate it, hitting the dock button places the lyrics in a top window,
not on the side where it can be sized to be longer.

It is Audacious 4.4 that is not displaying album art in my .lua scripted
NowPlaying widget. It was authored by a coder friend and uses mpris but not
playercrl. Audacious 4.1 displays the album art fine.
